src/pty.rs

```diff
@@ -16,78 +16,58 @@ use crate::{fcntl, unistd, Result};
 
 /// Representation of a master/slave pty pair
 ///
-/// This is returned by `openpty`.  Note that this type does *not* implement `Drop`, so the user
```

Comment:
Since we explicitly called this out in the documentation before, do you think we should warn people in the CHANGELOG that they no longer need to close the file descriptors?

Reply:
Yep, this should be documented, I will take care of it.

Reply:
Done.